travis/id_rsa

```diff
@@ -0,0 +1,27 @@
+-----BEGIN RSA PRIVATE KEY-----
```

Comment:
Relevant links:
    * https://developer.github.com/guides/managing-deploy-keys/
    * https://docs.travis-ci.com/user/encrypting-files/

See also how pushing to the `cabal-website` repo is implemented.